### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by moconnell**
## Summary by CodeRabbit

* **Tests**
  * Added comprehensive unit and integration tests across volatility calculations, ticker aliasing, API edge cases, type conversions, signing, mocked broker flows, and integration scenarios.

* **Chores**
  * Added and configured a new test project, updated CI report filters to ignore build artifacts, and exposed internals to tests for richer coverage.

* **Refactor / API**
  * Updated broker APIs to add an optional includeToday parameter to daily price retrieval, adjusting date-boundary behavior and caller signatures.

### CODE REVIEW

File: src/YoloBroker.Hyperliquid/HyperliquidBroker.cs [hunk 1]
```diff
@@ -814,11 +815,12 @@ private async Task<HyperLiquidOrderBook> GetFuturesOrderBookAsync(string symbol,
...
         int periods,
+        bool includeToday = false,
         CancellationToken ct = default)
     {
         ticker = _tickerAliasService.TryGetAlias(ticker, out var alias) ? alias! : ticker;
-        var startTime = DateTime.UtcNow.AddDays(-periods);
-        var endTime = DateTime.UtcNow;
+        var endTime = includeToday ? DateTime.UtcNow : DateTime.Today;
+        var startTime = DateTime.Today.AddDays(-periods + 1);
         var klines = await GetDataAsync(
```
[the remark this answers is not in the document]
Comment (moconnell) [reply src/YoloBroker.Hyperliquid/HyperliquidBroker.cs]: fixed
